File: tourneyAPI/Models/Entities/Characters/PiranhaPlant.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class PiranhaPlant : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public PiranhaPlant()
    {
        Id = CharacterId.PiranhaPlant;
        characterName = CharacterName.PIRANHA_PLANT;
        archetype = Archetype.TRAPPER;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.HEAVYWEIGHT;
        tierPlacement = TierPlacement.D;
    }

}

File: tourneyClient/src/models/entities/Characters/PiranhaPlant.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { Archetype } from "@/models/Enums/Archetype";
import { CharacterName } from "@/models/Enums/CharacterName";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const PiranhaPlant: Character =
{
    id: CharacterId.PiranhaPlant,
    characterName: CharacterName.PIRANHA_PLANT,
    archetype: Archetype.TRAPPER,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.HEAVYWEIGHT,
    tierPlacement: TierPlacement.D
}

export default PiranhaPlant; 